contrib/systemd/etcd2-backup-coreos/README.md

```diff
@@ -10,15 +10,15 @@ Remote backup and multi-node restore services for etcd2 clusters on CoreOS Linux
 
 ## Configuration
 
-Before installing etcd2-backup, you need to configure `30-etcd2-backup-restore.conf`.
+Before installing etcd2-backup, configure `30-etcd2-backup-restore.conf`:
 
 ```
 [Service]
 Environment="ETCD_RESTORE_MASTER_ADV_PEER_URLS=<http://host:port>"
 Environment="RCLONE_ENDPOINT=remote-name:path/to/backups"
 ```
 
-Assuming you're deploying to CoreOS with etcd2, you should only need to change
+Assuming a deployment to CoreOS with etcd2, only change:
```

Comment:
there are still a few places where the rebrand needs to be done; should defer to another patchset